editor/editor_log.cpp

```diff
@@ -62,6 +62,18 @@ void EditorLog::_notification(int p_what) {
 		//button->set_icon(get_icon("Console","EditorIcons"));
 		log->add_theme_font_override("normal_font", get_theme_font("output_source", "EditorFonts"));
 		log->add_theme_color_override("selection_color", get_theme_color("accent_color", "Editor") * Color(1, 1, 1, 0.4));
+		log->add_theme_font_override("bold_font", get_theme_font("bold", "EditorFonts"));
+
+		type_filter_map.find(MSG_TYPE_STD)->get()->toggle_button->set_icon(get_theme_icon("Popup", "EditorIcons"));
+		type_filter_map.find(MSG_TYPE_ERROR)->get()->toggle_button->set_icon(get_theme_icon("StatusError", "EditorIcons"));
+		type_filter_map.find(MSG_TYPE_WARNING)->get()->toggle_button->set_icon(get_theme_icon("StatusWarning", "EditorIcons"));
+		type_filter_map.find(MSG_TYPE_EDITOR)->get()->toggle_button->set_icon(get_theme_icon("Edit", "EditorIcons"));
```

Comment:
If we're sure that those four entries exist, maybe they can be indexed directly as done in other places?
```suggestion
		type_filter_map[MSG_TYPE_STD]->toggle_button->set_icon(get_theme_icon("Popup", "EditorIcons"));
		type_filter_map[MSG_TYPE_ERROR]->toggle_button->set_icon(get_theme_icon("StatusError", "EditorIcons"));
		type_filter_map[MSG_TYPE_WARNING]->toggle_button->set_icon(get_theme_icon("StatusWarning", "EditorIcons"));
		type_filter_map[MSG_TYPE_EDITOR]->toggle_button->set_icon(get_theme_icon("Edit", "EditorIcons"));
```